src/components/LeftContainer.js

```diff
@@ -90,6 +104,10 @@ const LeftContainer = ({
     setInput(e.target.value);
   };
 
+  const handleResetCode = () => {
+    localStorage.removeItem(mode);
```

Comment:
We can add a prompt here to ask the user:
This will reset the code.
If Yes - Proceed to clear local storage
else - Do nothing